common/static/common/templates/discussion/topic.underscore

```diff
@@ -6,7 +6,7 @@
         <%- gettext("Add your post to a relevant topic to help others find it. (Required)") %>
     </div>
     <div class="field-input">
-        <select class="post-topic field-input" aria-describedby="field_help_topic_area" required>
+        <select class="post-topic field-input" aria-describedby="field_help_topic_area wrapper-visibility-message" required>
```

Comment:
@smarnach this looks better, but it looks like the inclusion of the  #wrapper-visibility-message is conditional, but the reference to it in this aria-describedby is not. Unfortunately, if the aria-describedby refers to an non-existent IDref, it will produce an error.  Can you make this reference conditional as well, OR, you can just always render the #wrapper-visibility-message div, and only add content to it conditionally, that way it is always there, it just may be empty (that does not produce an error.)

Reply:
Ah never mind!  I see you did that here: https://github.com/edx/edx-platform/pull/15490/commits/f292abe7c59e9d742e9435fd7c3d971ee1812866